### DISCUSSION TITLE
Allow bows + flame to drop exp on mobs killed by the fire tick

### DISCUSSION THREAD

**Discussion by Decme** [PurpurMC/Purpur#897] [__Completed__] [answered]
Apparently when bows enchanted with flame shoot a mob, and the mob dies to the flames, the mob does not drop exp. Would be nice to have a toggle for this.

**Comment by BillyGalbreath** [comment, marked as answer]
There is an internal counter of 100 ticks for this. If the mob does not die within those 100 ticks of being hurt by a player (from a projectile or directly) then the kill does not count as a player kill. For exp to drop, the kill must count as a player kill.

**Reply by masmc05** [reply]
add a config to be able to expand those 100 ticks? (or maybe someone may want to decrease it)

**Reply by BillyGalbreath** [reply]
Implemented in build 1550

https://github.com/PurpurMC/Purpur/commit/314ca1a1bea24e71a54c9e79f4f73d6e2b463aff
